task/deprecated-image-check/0.4/deprecated-image-check.yaml

```diff
@@ -29,6 +29,8 @@ spec:
   results:
     - description: Tekton task test output.
       name: TEST_OUTPUT
+    - description: Images processed in the task.
+      name: IMAGES_PROCESSED
```

Comment:
Correct me if I'm wrong, but EC is expecting those results to be there? I'm trying to figure out what is the purpose of creating a new result.

Reply:
Yes, EC is expecting those results. The purpose of IMAGES_PROCESSED result is for EC to verify that all the image manifests of an image index were scanned by the task.